middleware/validate-activityContent.js

```diff
@@ -0,0 +1,11 @@
+const checkActivity = (request,response,next) => {
+    if (!request.body.name || !request.body.content || !request.file){
+        return response.status(200).send("NOT_ACCEPTABLE")
```

Comment:
`NOT_ACCEPTABLE` isn't imported and 200 should be replaced importing it from `constants/httpStatus`